scripts/start-local.ps1

```
@@ -0,0 +1,60 @@
+$venvActivate = Join-Path $repoRoot ".venv\Scripts\Activate.ps1"
+$localDataDir = Join-Path $repoRoot "stack\local-data"
+$ttlPath = Join-Path $localDataDir "data_model.ttl"
+$ttlMirrorPath = Join-Path $localDataDir "data_model.mirror.ttl"
+
+New-Item -ItemType Directory -Path $localDataDir -Force | Out-Null
+
+if (-not (Test-Path $venvActivate)) {
+  throw "Missing venv activation script: $venvActivate"
+}
+
+function New-ServiceCommand([string]$serviceCommand, [string]$cwd) {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'New-ServiceCommand' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)